src/components/MessageAttachment.vue

```diff
@@ -34,45 +34,51 @@
 			<span class="attachment-size">{{ humanReadable(size) }}</span>
 		</div>
 		<Actions :boundaries-element="boundariesElement">
-			<ActionButton
-				v-if="isCalendarEvent"
-				class="attachment-import calendar"
-				:disabled="loadingCalendars"
-				@click.stop="loadCalendars">
-				<template #icon>
-					<IconAdd v-if="!loadingCalendars" :size="20" />
-					<IconLoading v-else-if="loadingCalendars" :size="20" />
-				</template>
-				{{ t('mail', 'Import into calendar') }}
-			</ActionButton>
-			<ActionButton
-				class="attachment-download"
-				@click="download">
-				<template #icon>
-					<IconDownload :size="20" />
-				</template>
-				{{ t('mail', 'Download attachment') }}
-			</ActionButton>
-			<ActionButton
-				class="attachment-save-to-cloud"
-				:disabled="savingToCloud"
-				@click.stop="saveToCloud">
-				<template #icon>
-					<IconSave v-if="!savingToCloud" :size="20" />
-					<IconLoading v-else-if="savingToCloud" :size="20" />
-				</template>
-				{{ t('mail', 'Save to Files') }}
-			</ActionButton>
-			<div
```

Comment:
would it not be sufficient to move the div outside of the actions element?

Reply:
no, because popovermenu is deprecated as a component and the second calendar popover is shown behind the "body" of the event.